### ISSUE TITLE
Start fails when running in windows users' home

### ISSUE THREAD

**Issue by pofider** [jsreport/jsreport#408]
This should be fixed or there should be a better error message.

```js
C:\Users\janbl>jsreport start
2018-05-17T12:43:58.740Z - info: Initializing jsreport@2.0.0 in development mode using configuration file: jsreport.config.json
2018-05-17T12:43:58.743Z - info: Searching for available extensions in C:\Users\janbl\
2018-05-17T12:43:58.752Z - info: Extensions location cache not found, crawling directories
2018-05-17T12:43:58.764Z - error: Error occured during reporter init TypeError: Cannot read property 'forEach' of undefined
    at Object.exports.walkSync (C:\Users\janbl\node_modules\jsreport-core\lib\util\util.js:46:10)
    at Object.get (C:\Users\janbl\node_modules\jsreport-core\lib\extensions\locationCache.js:22:22)
    at <anonymous>
Error: An error has occurred when trying to initialize jsreport..
    at instance.init.then.catch (C:\Users\janbl\node_modules\jsreport-cli\lib\instanceHandler.js:31:29)
    at <anonymous> {"originalError":{}}
(Original) TypeError: Cannot read property 'forEach' of undefined
    at Object.exports.walkSync (C:\Users\janbl\node_modules\jsreport-core\lib\util\util.js:46:10)
    at Object.get (C:\Users\janbl\node_modules\jsreport-core\lib\extensions\locationCache.js:22:22)
    at <anonymous> {}
```

**Comment by gabsschneider** [comment]
I am having the same issue

**Comment by pofider** [comment]
The workaround is to run install in dedicated directory like 
`c:\work\app`

**Comment by pofider** [comment]
Fixed in 2.1.0
